confluent_platform/assets/monitors/unclean_leader_election.json

```diff
@@ -2,11 +2,11 @@
   "version": 2,
   "created_at": "2021-10-28",
   "last_updated_at": "2021-10-28",
-  "title": "Unclean leader election",
+  "title": "Unclean leader election detected",
   "tags": [
     "integration:confluent-platform"
   ],
-  "description": "Get notified if an unclean leader election has taken place.",
+  "description": "The unclean leader election setting decides whether a trailing replica might take over as the leader in case of failures. However, appointing an unsynchronized replica as leader risks message loss for any data it hasn't yet replicated. This monitor tracks unclean leader elects for each individual instance.",
```

Comment:
```suggestion
  "description": "The unclean leader election setting decides whether a replica might take over as the leader in case of failures. However, appointing an unsynchronized replica as leader risks message loss for any data it hasn't yet replicated. This monitor tracks unclean leader elects for each individual instance.",
```